scss/lessonCard.module.scss

```diff
@@ -13,7 +13,7 @@
   padding-right: 1em;
   margin-left: auto;
   @media (max-width: 768px) {
-    background-color: transparent;
+    background-color: transparent !important;
```

Comment:
`!important` is bad practice, if you must use it it should be accompanied with a comment explaining why.

Reply:
did you try removing the `.bg-success` class?
<img width="723" alt="Screen Shot 2022-02-16 at 5 39 51 AM" src="https://user-images.githubusercontent.com/686933/154276399-404952d0-8112-4068-b40c-80943a0ac6b3.png">

Reply:
Removing it for mobile and tablet fixes it but not desktop. One solution is to separate tablet and desktop styles but I chose to set its default bg to success and change it to transparent when the media query condition is true.

Now the flag isn't needed.